### DISCUSSION TITLE
Native billing module

### DISCUSSION THREAD

**Discussion by purejosh** [pelican-dev/panel#41] [Ideas]
I know this is a long shot, but the main issue I've had with Pterodactyl is its complete and total lack of a native billing module. Even if it were something simple, like "this egg with these resources bills at this much a month" with basic paypal integration, I think you'd be leaps and bounds above what Ptero could ever hope for. Especially with all of the billing modules for Ptero being paid. 

It's not a simple ask. But it would be incredible to see. Paypal can charge paypal accounts, and credit cards without a sign in. Using a credit balance system (meaning you purchase credits, and your server will cost you x amount of credits per hour) would feel most natural, though a monthly billing and proration if terminated prematurely would also work.

**Comment by lancepioch** [comment]
Yup, we are considering a first party plugin or modification regarding this!

**Reply by WajPad** [reply]
yes

**Comment by Craftefix** [comment]
hows the status on this @lancepioch

**Reply by rmartinoscar** [reply]
Won't be before full release for sure.
